internal/input/orderbooks.go

```diff
@@ -0,0 +1,258 @@
+package input
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"sync"
+
+	ingestio "github.com/stellar/go/ingest/io"
+	"github.com/stellar/go/ingest/ledgerbackend"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/stellar-etl/internal/transform"
+)
+
+type OrderbookBatch struct {
+	BatchStart uint32
+	BatchEnd   uint32
+	Orderbooks map[uint32][]ingestio.Change
+}
+
+// OrderbookParser handles parsing orderbooks
+type OrderbookParser struct {
```

Comment:
If there's any official Stellar documentation relevant to understanding this data structure (i.e., structure of orderbook), it would be great to reference that here in a comment.

Reply:
There isn't any documentation right now. I'll open another PR to update the readme with the exact schema/definition.